packages/twenty-front/src/modules/object-record/record-table/components/RecordTable.tsx

```diff
@@ -141,6 +157,7 @@ export const RecordTable = ({
         >
           <StyledTable
             freezeFirstColumns={scrollLeft > 0}
+            freezeHeaders={scrollTop > 0}
```

Comment:
if I set freezeHeaders to false will it not freeze headers? Seems like you don't re-use this?

Reply:
I was unsure about the style changes so the state was already implemented, I created a variable so that it can be used for conditional styling like freezed first column.

Reply:
It's a great idea to introduce it, but I think it's confusing to introduce the variable but not make it work though. If we introduce it then we should be able to disable sticky header by passing false if we re-use it in another context. 
` freezeFirstColumns` is used to adapt the CSS conditionally. If we'd like to keep this var (which would be nice) we should do the same

Reply:
Yeah I also thought same but there's no shadow for freezed headers and no other design change. Should I add similar design to freezed header as in freezed column